src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,9 +7,8 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 1102;
```

Comment:
The goal for the root and `issues` is to be empty, so we need to keep the restrictions, even if they are less than 1000.